client/app.js

```diff
@@ -122,6 +122,20 @@ tie.constant('SYSTEM_CODE', {
   ].join('\n')
 });
 
+/**
+ * Provides class names for system and auxiliary code according to the language
+ * that is being used.
+ *
+ * @type {{}}
+ * @constant
+ */
+tie.constant('SYSTEM_AUXILIARY_CLASSES', {
```

Comment:
I thought we already had constants for these (cf one of Elena's recent PRs). I think it's also fine for them to be consistent across all languages?